docs/source/1.0/spec/core/constraint-traits.rst

```diff
@@ -556,6 +556,48 @@ in a response.
         }
 
 
+.. _sparse-trait:
```

Comment:
I wasn't sure if this should be a "constraint trait" or a "type refinement trait"...

Reply:
Feels like this should be a "type refinement trait" as it's dealing in the same realm as `@box` which lives there as well.